exporters/otlp/build.gradle

```diff
@@ -18,7 +18,9 @@ dependencies {
             libraries.protobuf,
             libraries.protobuf_util
 
-    testImplementation project(':opentelemetry-sdk-testing')
+    testImplementation project(':opentelemetry-sdk-testing'),
+            'com.linecorp.armeria:armeria-grpc',
+            'com.linecorp.armeria:armeria-junit5'
```

Comment:
Why this change?

Reply:
yeah, feels like this snuck in to the wrong PR